src/node.cc

```diff
@@ -228,16 +228,31 @@ static struct {
   }
 #endif  // HAVE_INSPECTOR
 
+  void StartTracingAgent() {
+    tracing_agent = new tracing::Agent();
```

Comment:
Can you make this a property of `v8_platform` while you're here (and call it `tracing_agent_`)?  Maybe also insert a `CHECK(tracing_agent_ == nullptr)` to detect double calls.